### ISSUE TITLE
Explainer or more examples?

### ISSUE THREAD

**Issue by bfgeek** [w3c/webauthn#330]
So while trying to understand this API I needed to open up Edge's guide to the slightly older API.
https://developer.microsoft.com/en-us/microsoft-edge/platform/documentation/dev-guide/device/web-authentication/

Is an explainer / examples in the github which have more fleshed out examples (like in the above link possible)?

Somewhat difficult to understand with just the client snippet, not a lot of context. :)

**Comment by sideshowbarker** [comment]
Possible duplicate of https://github.com/w3c/webauthn/issues/91 (which was created in May of last year but then closed by @bifurcation without comment in September…)

**Comment by nadalin** [comment]
Thanks for the comments, this is currently out of scope for us to create as part of our charter and the specification needs to remain as generic as we can. Maybe someone will create a best practice or usage guide once the specification is complete

**Comment by sideshowbarker** [comment]
> this is currently out of scope for us to create as part of our charter

That assertion isn’t true. The group’s charter does not prohibit the group, as part of its work, from publishing non-normative/informative materials (such as an explainer doc) to accompany the normative deliverables it is chartered to publishing. No W3C charter prohibits that.

But the group is of course not obligated to publish any informative materials like this. However if there’s somebody in the group who actually has an interest in volunteering to author an explainer, it would certainly not be consistent with W3C policy/process for the group block them.

The point of this issue and the one I filed is: We look to the members of this group for the expertise about their own spec—subject-matter expertise on their own work sufficient to explain the work at a high level to others who are not subject-matter experts. That’s all the being asked for here.

Of course if nobody in the group has interest and time to write an explainer, than it’s be find to close this with a statement saying that. But it’s not fine to claim it can’t be done because the group’s charter prevents the group from working on it and publishing it in some form. Because that’s not true.

**Comment by nadalin** [comment]
Sorry but it is, we have a clear charter with deliverables and dates to meet this is not in our charter. Charters and deliverables are created for a reason. We discussed this issue on the call yesterday and we chose close this as out of scope. If you wish to join the group and write the document feel free to as we have no resources or time to produce such a document, we all believe that the developer community will create a document if needed
